app/config/config_dev.yml

```diff
@@ -27,7 +27,7 @@ monolog:
         server_log:
             type:   server_log
             process_psr_3_messages: false
-            host: 0:9911
+            host: localhost:9911
```

Comment:
Yes. I guess I wasn't sure what the implication of changing it to localhost would be. I'm guessing 0 gets translated to 0.0.0.0 on 'nix.